watcher/tool/watcher/main.go

```diff
@@ -54,23 +86,18 @@ func main() {
 	case constants.ModeDashboards:
 		err = runDashboardsWatcher(ctx, client, retryC)
 	case constants.ModeRollups:
-		err = runRollupsWatcher(ctx, client, retryC)
+		err = runRollupsWatcher(ctx, client, influxDBConfig, retryC)
 	case constants.ModeAlerts:
 		err = runAlertsWatcher(ctx, client, retryC)
 	default:
-		fmt.Printf("ERROR: unknown mode %q\n", mode)
-		os.Exit(255)
+		return trace.Errorf("ERROR: unknown mode %q\n", mode)
```

Comment:
`trace.BadParameter` and avoid `ERROR` prefix since it might distort the error message (i.e. if the caller also adds it).